### PR TITLE
Community event layout

### PR DESCRIPTION
**Pull request by IMightBeAce**

### CODE REVIEW

File: views/js/evently/src/CommunityPage.jsx [hunk 1]
```diff
@@ -1,11 +1,172 @@
...
+       }
+     }
+     catch (err){
+       console.log(err);
+     }
+   }
+   useEffect(()=>{
+    getEvent()
+   }, [])
+   console.log(eventData)
```
Comment (mclausen792) [inline views/js/evently/src/CommunityPage.jsx:26]: You can probably remove this console.log before merge

**Review by mclausen792 — APPROVED** [review]
Just one little comment. It is typically better to remove console.logs before merge. Fine to leave the error one since this is not going to be a production build. Good job!

File: views/js/evently/src/CommunityPage.jsx (hunk 1 above)
Comment (IMightBeAce) [reply views/js/evently/src/CommunityPage.jsx:26]: I see thank you. In the future should I delete lines 23-26 or just line 26?

Comment (mclausen792) [reply views/js/evently/src/CommunityPage.jsx:26]: Just 26!

**Pull request merged**